### DISCUSSION TITLE
Nuxt server keeps restarting when anything on client changes

### DISCUSSION THREAD

**Discussion by khalednadam** [nuxt/nuxt#24128] [Questions] [answered]
My Nuxt 3 server keeps restarting every time I change anything on the client. I don't know if it is something related to the config but here is my config file

I tried removing the import and I still have the same issue

`import { myTheme } from "./vuetify.config.js";
export default defineNuxtConfig({
    devtools: { enabled: true },
    nitro: {
        routeRules: {
            // ... routes rules
        },
    },
    modules: [
        "@invictus.codes/nuxt-vuetify",
        "@nuxtjs/tailwindcss",
        "@nuxtjs/i18n",
        "@nuxtjs/google-fonts",
        "@vueuse/motion/nuxt",
        "@vee-validate/nuxt",
        "@nuxt/image",
    ],
    tailwindcss: {
        cssPath: "~/assets/css/tailwind.css",
    },
    css: ["~/assets/main.scss"],
    googleFonts: {
        // ... fonts options
    },
    image: {
        dir: "assets/",
    },
    i18n: {
        vueI18n: "./i18n.config.js",
        // ... i18n options
    },
    vuetify: {
        vuetifyOptions: {
            // ... vuetify options
            // USES imported theme { myTheme }
        },
    },
});`

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Reply by khalednadam** [reply]
https://stackblitz.com/edit/github-f2wmlk?file=app.vue

I don't know if it isn't working due to the same reason.

`
[5:45:18 PM]  ERROR  (node:134) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use node --trace-warnings ... to show where the warning was created)


[5:45:18 PM]  ERROR  Cannot start nuxt:  Cannot read properties of undefined (reading 'options')
`

**Reply by khalednadam** [reply]
by the way, each time it restarts the server it recreats the components directory

**Reply by TheAlexLichter** [reply]
> I don't know if it isn't working due to the same reason.

Apparently the server doesn't even start there because nuxt i18n uses the non-nuxt-3 version (should be 8.X beta)

**Comment by khalednadam** [comment, marked as answer]
After long hours of trying to understand what is causing the problem. My nuxt config was .ts and I think I have problem setting up typescript so I think that what was causing the issue
